cmd/influxd/run.go

```diff
@@ -272,15 +376,21 @@ func parseConfig(path, hostname string) (*Config, error) {
 }
 
 // creates and initializes a broker.
-func openBroker(path string, u url.URL, initializing bool, joinURLs []url.URL, raftTracing bool) (*influxdb.Broker, *raft.Log) {
+func (cmd *RunCommand) openBroker(brokerURLs []url.URL) {
```

Comment:
Oh, yes, I think this is all very strange. Again, I might be missing something, but it seems to me `RunCommand` should have returned a `Node` and `Node` would have these methods like `openBroker`.